### PR TITLE
Fixed golint issues in maya/pkg/nethelper/ipcalc.go

### PR DESCRIPTION
**Pull request by Axle7XStriker**
Fixed all the golint issues that were there in the `maya/pkg/nethelper/ipcalc.go`.

fixes openebs/openebs#2125
Signed-off-by: Axle7XStriker <amanbansal1997@gmail.com>

**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: pkg/nethelper/ipcalc.go [hunk 1]
```diff
@@ -1,6 +1,8 @@
-// This is an adaptation of below gist:
-//
-//  https://gist.github.com/kotakanbe/d3059af990252ba89a82
+/*
```
Comment (princerachit) [inline pkg/nethelper/ipcalc.go]: Undo this change and just add an empty line above `package nethelper`

Comment (Axle7XStriker) [reply pkg/nethelper/ipcalc.go]: Made the changes please see to it.
